types/grasp/package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Remove this file. You will depend on `@types/cli-color` and `@types/estree`, not on `cli-color` and `estree`.

Reply:
Ah, will that all happen automagically?

Reply:
Yes. For testing we have [path mapping](http://www.typescriptlang.org/docs/handbook/module-resolution.html) in the tsconfig.json. When publishing we will add the dependencies automatically.

Reply:
Sweet!